cinnabar/tests/test_stats.py

```diff
@@ -145,3 +145,47 @@ def test_correlation_positive(fe_map):
         assert (
             0.5 < bss["mle"] < 0.9
         ), f"Correlation must be positive for this data. {stat} is {bss['mle']}"
+
+
+def test_confidence_intervals(fe_map):
+    """
+    Test that the bootstrapped confidence intervals
+    for RMSE and MUE contain the corresponding statistics.
+    Uses the example data in `cinnabar/data/example.csv`
+    """
+
+    nodes = fe_map.graph.nodes
+
+    x_data = np.asarray([n[1]["exp_DG"] for n in nodes(data=True)])
+    y_data = np.asarray([n[1]["calc_DG"] for n in nodes(data=True)])
+    xerr = np.asarray([n[1]["exp_dDG"] for n in nodes(data=True)])
+    yerr = np.asarray([n[1]["calc_dDG"] for n in nodes(data=True)])
+
+    # RMSE (default mode)
+    bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="RMSE")
+    assert bss['low'] < bss['mle'] < bss['high'], \
+        "The RMSE must lie within the bootstrapped 95% CI"
+
+    # MUE (default mode)
+    bss = bootstrap_statistic(x_data, y_data, xerr, yerr, statistic="MUE")
```

Comment:
@IAlibay : I'm not familiar with `pytest.parametrize` -- would you have time to re-format this with `pytest.parametrize`? If not, maybe we could just merge this PR as is and I could open an issue about improving the tests in the long term? While I'm sure your suggestion would improve the code, I'm not sure its essential, as my goal here was just to introduce the minimal changes necessary so that I (and others in the Chodera lab) can continue making progress on the scientific side of things.

Reply:
Done